kibbeh/src/ui/ChangeAvatarCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ChangeEvent } from "react";
+import { BaseSettingsItem } from "./BaseSettingsItem";
+import { SingleUser } from "./UserAvatar";
+import { Button } from "./Button";
+import SolidTrashIcon from "../icons/SolidTrash";
+
+export interface ChangeAvatarCardProps {
+   avatarUrl: string
+}
+
+export const ChangeAvatarCard: React.FC<ChangeAvatarCardProps> = ({
+   avatarUrl
+}) => {
+   const handleImageUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
+      const MAX_FILE_SIZE = 3145728;
+      const file = (e.target.files as FileList)[0]
+
+      if(file.size > MAX_FILE_SIZE) return // show error toast
+
+      // handle upload
+      console.log(file);
+   }
+
+   const handleImageDelete = () => {
+      // handle delete
+   }
+
+   return (
+      <BaseSettingsItem className="flex items-center px-4 py-3">
+         <div>
+            <SingleUser src={avatarUrl} />
+         </div>
+         <div className="flex flex-col ml-5">
+            <div className="flex items-center">
+               <Button size="small" color="secondary" className="">
+                  <label htmlFor="avatar" className="relative cursor-pointer">
+                     Change profile picture
```

Comment:
This should be `text-primary-100`

Reply:
You mean on the label? If yes, the button already has that style

Reply:
Yeah, it's white rn

Reply:
Ok. done